nsone/rest/data.py

```diff
@@ -5,6 +5,20 @@
 #
 from . import resource
 
+class Job(resource.BaseResource):
```

Comment:
Would you mind moving this class to the [rest monitoring module](https://github.com/ns1/nsone-python/blob/master/nsone/rest/monitoring.py)?

And what if we called this class `Monitors` as opposed to `Job` to keep with the convention of [Records](https://github.com/ns1/nsone-python/blob/master/nsone/rest/records.py#L9) and [Zones](https://github.com/ns1/nsone-python/blob/master/nsone/rest/zones.py#L10)?